src/Homebrew.jl

```diff
@@ -77,7 +86,11 @@ function install_brew()
     if !isfile(joinpath(brew_prefix,"bin","otool"))
         # Download/install packaged install_name_tools
         try
-            run(`curl --location $BOTTLE_SERVER/cctools_bundle.tar.gz` |> `tar xz -C $(joinpath(brew_prefix,"bin"))`)
+            if use_pipeline
+                run(pipeline(`curl --location $BOTTLE_SERVER/cctools_bundle.tar.gz`, `tar xz -C $(joinpath(brew_prefix,"bin"))`))
```

Comment:
at least this one should work without the if because pipeline should be in Compat.jl - the other ones might be tuple destructuring related or something?